simulation/costs.py

```diff
@@ -345,7 +345,7 @@ def estimate_extended_boiler_upgrade_scheme_grant(
         return 0
 
     model_population_scale = ENGLAND_WALES_HOUSEHOLD_COUNT_2020 / model.household_count
-    boiler_upgrade_funding_cap_gbp = 450_000_000 / model_population_scale
+    boiler_upgrade_funding_cap_gbp = 1_950_000_000 / model_population_scale
```

Comment:
The context for this revision isn't in the PR description.

Reply:
Added context to the PR description. Also adjusted funding cap to 1_650_000 since we are now starting the model in 2024 r.t. 2022.